src/core/dbus-execute.c

```diff
@@ -692,6 +692,42 @@ static int property_get_log_extra_fields(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_path_images(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        ExecContext *c = userdata;
+        unsigned i;
+        int r;
+
+        assert(bus);
+        assert(c);
+        assert(property);
+        assert(reply);
+
+        r = sd_bus_message_open_container(reply, 'a', "(ssbs)");
+        if (r < 0)
+                return r;
+
+        for (i = 0; i < c->n_path_images; i++) {
+                r = sd_bus_message_append(
+                                reply, "(ssbs)",
+                                c->path_images[i].source,
+                                c->path_images[i].destination,
+                                c->path_images[i].ignore_enoent,
```

Comment:
hmm, i wonder if only masking this one failure explicit is ideal. maybe call this "permissive" and make it a bool that means "ignore any failures of this mount regardless what they are"?

Reply:
ok